psi4/driver/driver_findif.py

```diff
@@ -0,0 +1,799 @@
+from __future__ import division
+from psi4.driver.p4util.exceptions import ValidationError
+from psi4.driver.qcdb import libmintsmolecule
```

Comment:
Help me keep libmintsmolecule hidden, please. Perhaps try it import it from molecule instead (https://github.com/psi4/psi4/blob/master/psi4/driver/qcdb/__init__.py#L40) or failing that just move it to molecule.py or add it as a staticmethod to qcdb.Molecule.